docs/libraries/mst-form-type.md

```diff
@@ -0,0 +1,95 @@
+# MST Form Type
+
+npm: https://www.npmjs.com/package/mst-form-type
+
+## Introduction
+
+Librarys like Ant Design has a build-in Form component that can hold fields status and validation rules. This component houses its own store to sustain internally related field statuses and can be conveniently used independently. integrating Form component with MobX State Tree models can pose significant challenges as business logic grows intricately complex.
+
+The `mst-form-type` library comes into play. It provides the field management feature of Ant Design Form component in Mobx State Tree model way. Users can only use the UI of a component library, but keep logic inside your Mobx State Tree model context, instead of syncing status changes between Form component and model.
```

Comment:
```suggestion
That's where a solution like `mst-form-type` library comes into play. It models the Ant design field management like a conventional MobX-State-Tree type definition. Users can still use the UI of a component library and keep logic inside MobX-State-Tree, instead of syncing status changes between `Form` component and model.
```

**suggestion:** I reworded a few things and fixed spelling/grammar errors here.